templates/types/streaming/fastapi/app/llmhub.py

```diff
@@ -0,0 +1,46 @@
+from llama_index.embeddings.openai import OpenAIEmbedding
```

Comment:
it's a new file - is it copied by the code generator?

Reply:
yes it is copied in backend.